src/item.cpp

```diff
@@ -3402,6 +3414,10 @@ std::string item::tname( unsigned int quantity, bool with_prefix, unsigned int t
         ret << _( " (UPS)" );
     }
 
+    if( is_tool() && has_flag( "USE_UPS_AIR" ) ) {
+        ret << _( " (UPS_AIR)" );
+    }
```

Comment:
This is probably not needed at all. And even if it is, it shouldn't display internal game values, it should display user-friendly values like `(air cylinder)`.

Reply:
I guess not. Tested the firing of a pellet with the modified gun and seems to work.